CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ Mapbox welcomes participation and contributions from everyone.
 
 ### main
 
+### v7.2.0 - August 28, 2024
+
+- Added `PaymentMethods#etc2` payment method.
+- Added `V6StructuredInputQuery#country` option.
+
+
+### v7.1.0 - July 8, 2024
```

Comment:
Should it be there?

Reply:
Yes, it was released, but changelog has not been updated https://github.com/mapbox/mapbox-java/releases/tag/v7.1.0